e2e/angular/src/module-federation.rspack.test.ts

```diff
@@ -43,23 +43,23 @@ describe('Angular Module Federation', () => {
     runCLI(
       `generate @nx/angular:host ${hostApp} --style=css --bundler=rspack --no-standalone --no-interactive`
     );
-    let rspackConfigFileContents = readFile(join(hostApp, 'rspack.config.ts'));
-    let updatedConfigFileContents = rspackConfigFileContents.replace(
-      `maximumError: '1mb'`,
-      `maximumError: '11mb'`
-    );
-    updateFile(join(hostApp, 'rspack.config.ts'), updatedConfigFileContents);
+    // let rspackConfigFileContents = readFile(join(hostApp, 'rspack.config.ts'));
+    // let updatedConfigFileContents = rspackConfigFileContents.replace(
+    //   `maximumError: '1mb'`,
+    //   `maximumError: '11mb'`
+    // );
+    // updateFile(join(hostApp, 'rspack.config.ts'), updatedConfigFileContents);
 
     // generate remote app
     runCLI(
       `generate @nx/angular:remote ${remoteApp1} --host=${hostApp} --bundler=rspack --port=${remotePort} --style=css --no-standalone --no-interactive`
     );
-    rspackConfigFileContents = readFile(join(remoteApp1, 'rspack.config.ts'));
-    updatedConfigFileContents = rspackConfigFileContents.replace(
-      `maximumError: '1mb'`,
-      `maximumError: '11mb'`
-    );
-    updateFile(join(remoteApp1, 'rspack.config.ts'), updatedConfigFileContents);
+    // rspackConfigFileContents = readFile(join(remoteApp1, 'rspack.config.ts'));
+    // updatedConfigFileContents = rspackConfigFileContents.replace(
+    //   `maximumError: '1mb'`,
+    //   `maximumError: '11mb'`
+    // );
+    // updateFile(join(remoteApp1, 'rspack.config.ts'), updatedConfigFileContents);
```

Comment:
Should we remove this?

Reply:
I will - I just want to verify with MF that the budget does not need to change first because the MF runtime will be injected into the entrypoint(s)